src/containers/layout/navbar/NavBar.styles.js

```diff
@@ -16,7 +16,10 @@ export const styles = {
   },
   navItemText: {
     color: 'primary.900',
-    textDecoration: 'none'
+    textDecoration: 'none',
+    '&:focus': {
```

Comment:
I`m not sure is focus well in this case, because after scrolling out of the section, the button continues to be highlighted, we need to discuss this in meeting, I think hover would be better here

Reply:
> I`m not sure is focus well in this case, because after scrolling out of the section, the button continues to be highlighted, we need to discuss this in meeting, I think hover would be better here

We discussed it. Made some changes 👍